src/main/kotlin/calculator/view/InputView.kt

```diff
@@ -0,0 +1,10 @@
+package calculator.view
+
+private const val INPUT_TEXT = "구분자와 함께 덧셈할 숫자를 입력하세요"
+
+object InputView {
+    fun getInput(): String? {
+        println(INPUT_TEXT)
+        return readlnOrNull()
```

Comment:
```suggestion
    return readln()
```

readln()을 사용하여 null을 방지하는 건 어떨까요?

참고 
- [kotlin readln](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.io/readln.html)
- [kotlin 1.6.0 released](https://blog.jetbrains.com/ko/kotlin/2021/12/kotlin-1-6-0-is-released/)

Reply:
해당 부분, 요구사항에 null이 입력되면 0을 반환한다. 라는 부분이 있어 null도 입력가능하게 하였습니다!